### ISSUE TITLE
targetItems.csv and targetUsers.csv

### ISSUE THREAD

**Issue by abdollahpouri** [recsyschallenge/2017#14]
Hello there,
So, we have the interactions.csv file which is our training set and the test set is the one that you evaluate our submissions based on that. 

My question is about two files: targetItems.csv and targetUsers.csv. Are the items in targetItems.csv the same as the items in the test set? I mean does this file have the exact same items or maybe it has some more or maybe fewer items than the items in the test set. I have the same question about targetUsers.csv. 

Thank you very much for your cooperation,
Himan

**Comment by fabianabel** [comment]
Hi Himan, 

- all IDs in targetItems.csv point to an item in items.csv 
- all IDs in userTargets.csv point to a user in users.csv
- when submitting solutions: 
  + for each ID in targetItems.csv, you mention in your solution file those users to whom you would recommend/push the item
  + only users that appear in userTargets.csv should appear in your solution file (i.e. userTargets.csv is a hint from us that makes your life easier: no need to select from all users but just from the targetUsers.csv)

Cheers,
fabian

**Comment by abdollahpouri** [comment]
Thanks, Fabian,

Maybe I didn't explain my question very good. 

The reason for my question is to find out how exactly you created targetItems.cvs and targetUsers.csv so that we could do the same when we make our own test set. I know that target Items and target users both have entries in items.csv and users.csv, otherwise the task would have been impossible to solve. But, are target items exactly those that appear in the test set? (by test set I mean the last one-week interaction data) or maybe targetItems.csv contains more items or maybe fewer items? For example, one of thing about target items is that they do not exist in the train set (i.e. there is no interaction for those items in the training set) Am I right?
The same question holds for users in target users.

Thanks again,
Himan

**Comment by fabianabel** [comment]
Target items are items that have been created after the training period, i.e. this challenge is about recommending new items that have not received any interaction at the time the recommendations should be rolled out and the target items are exactly the items in the test set for which recommendations should be computed. 

Regarding target users: this is a randomly selected subset of users who were active on XING in the test period.
